lib/frontend/completeRegistration.js

```diff
@@ -30,6 +33,20 @@ module.exports = async (req, res, next) => {
       appId: result.id,
       githubHost: new URL(result.html_url).hostname
     });
+
+    // Update the webhook secret
+    const hostname = new URL(result.html_url).hostname;
```

Comment:
does this "hostname" and line no 15 gitHubHost are not same ?

Reply:
Yes, we should be reuse the hostname now. Couldn't use it before because logic in line #15 had to be updated.